### PR TITLE
Week 6 email: Linux gaming

### PR DESCRIPTION
**Pull request by jrtechs**
Drafted this week's email

### CODE REVIEW

File: announcements/_posts/2020-02-21-w06-linux-gaming.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Hey RITluggers!
+
+This week Dev is giving a talk on the exciting developments in Linux gaming. 
+
+After years of anticipation, 
+Linux finally has some traction in the game dev industry. 
+Since 2013, things have only gotten exponentially better for gaming on Linux.
+So what does Linux gaming look like now, 
+and what developments are on the horizon?
+Come to this week's meeting to become an Ebic LINUX Gaimer!
```
Comment (Tjzabel) [inline announcements/_posts/2020-02-21-w06-linux-gaming.md]: _oh no_

File: announcements/_posts/2020-02-21-w06-linux-gaming.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey RITluggers!
+
+This week Dev is giving a talk on the exciting developments in Linux gaming. 
+
+After years of anticipation, 
+Linux finally has some traction in the game dev industry. 
+Since 2013, things have only gotten exponentially better for gaming on Linux.
```
Comment (Tjzabel) [inline announcements/_posts/2020-02-21-w06-linux-gaming.md]: What's with the indentation of splitting up the sentence? And lines 15/16 are kinda the same thing. Just worded a little odd.

File: announcements/_posts/2020-02-21-w06-linux-gaming.md (hunk 1 above)
Comment (Tjzabel) [reply announcements/_posts/2020-02-21-w06-linux-gaming.md]: I really don't know how I feel about this sentence lol

Comment (jrtechs) [reply announcements/_posts/2020-02-21-w06-linux-gaming.md]: It's iconic :1st_place_medal:

File: announcements/_posts/2020-02-21-w06-linux-gaming.md (hunk 2 above)
Comment (jrtechs) [reply announcements/_posts/2020-02-21-w06-linux-gaming.md]: I'm gonna drop line 14

**Pull request merged**